Model/updatePosts.php

```diff
@@ -0,0 +1,48 @@
+
+<?php
+session_start();
```

Comment:
fix indentation

Reply:
It's something gitHub or push issue , because its proper in my code.